pkg/startup/v3/startup.go

```diff
@@ -24,16 +24,19 @@ import (
 	"github.com/openshift/openshift-azure/pkg/util/writers"
 )
 
-type startup struct {
+// Startup is the main configuration struct for startup
+type Startup struct {
```

Comment:
don't make this exported - I specifically want it unexported so that callers are forced to use startup.Interface, which this struct implements